package.json

```diff
@@ -65,19 +65,19 @@
   },
   "devDependencies": {
     "ava": "^1.4.1",
-    "chalk": "^1.1.3",
+    "chalk": "^4.1.2",
     "coffee-script": "^1.12.7",
-    "eslint": "^5.16.0",
-    "eslint-config-airbnb-base": "^13.1.0",
-    "eslint-plugin-import": "^2.17.3",
+    "eslint": "^8.2.0",
```

Comment:
Please remove these dependency changes from your PR. I would like your PR to be small and focused: only changes 1 thing to accomplish 1 goal. This PR should be only your feature (adding `.code` to the exception) and the unit test (`test/config.js`).

I can't merge your PR if it's doing all the things (updating dependencies, dropping nodejs version support, **and also** adding  a new feature + test). If you can rebase on top of the latest master branch and then `git push --force-with-lease` to your feature branch, then that will at least fix some of the errors you encountered on tip-of-tree.